cni/network/invoker_cns.go

```diff
@@ -169,8 +169,8 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 		// Do we want to leverage this lint skip in other places of our code?
 		key := invoker.getInterfaceInfoKey(info.nicType, info.macAddress)
 		switch info.nicType {
-		case cns.DelegatedVMNIC:
-			// only handling single v4 PodIPInfo for DelegatedVMNIC at the moment, will have to update once v6 gets added
+		case cns.NodeNetworkInterfaceFrontendNIC, cns.NodeNetworkInterfaceAccelnetFrontendNIC:
```

Comment:
this might break current working as cns is still sending delegatedvmnic - L205-214 https://github.com/Azure/azure-container-networking/blob/master/cns/middlewares/k8sSwiftV2.go. can we test this once on a non-vf setup?

Reply:
there is no impact since cns.DelegatedVMNIC is the same value as cns.NodeNetworkInterfaceFrontendNIC; changing from CNI will not impact the result; CNS would get impacted.

Reply:
isn't delegatedvmnic same as cns.NodeNetworkInterfaceFrontendNIC; I think from CNI side, it won't impact on any functionalities; I did not change CNS or DNC.

Reply:
we are checking for "nicType" value & not the string match here so there is no case for cns.DelegatedVMNIC which is being sent by cns. we should test this once with this branch's cni binary to make sure we don't break the flow